src/ammo.worker.js

```diff
@@ -152,6 +154,8 @@ const tick = () => {
 
     releaseBuffer();
   }
+
+  setTimeout(tick, simulationRate - stepDuration);
```

Comment:
Not sure I condone wrapping things in a try/catch since I think there is a big perf hit for that, but its worth noting that if any exceptions happen in tick the next tick will never be scheduled. It also may make sense to move this up to right after `stepDuration` is calculated (both for less chance of exceptions and more accurate scheduling, though I suspect setTimeout only offers so much precision anyway)